neural_search/README.md

```diff
@@ -192,103 +261,581 @@ Running sparse-search                                                          [
+|                                        90th percentile latency |                match-all |    7.42068 |     ms |
+|                                        99th percentile latency |                match-all |    10.7161 |     ms |
+|                                       100th percentile latency |                match-all |    13.9326 |     ms |
+|                                   50th percentile service time |                match-all |    5.90948 |     ms |
+|                                   90th percentile service time |                match-all |    6.71041 |     ms |
+|                                   99th percentile service time |                match-all |    9.36623 |     ms |
+|                                  100th percentile service time |                match-all |     11.778 |     ms |
+|                                                     error rate |                match-all |          0 |      % |
+|                                                 Min Throughput |        multimodal-search |       2.96 |  ops/s |
+|                                                Mean Throughput |        multimodal-search |       3.15 |  ops/s |
+|                                              Median Throughput |        multimodal-search |       3.15 |  ops/s |
+|                                                 Max Throughput |        multimodal-search |       3.31 |  ops/s |
+|                                        50th percentile latency |        multimodal-search |      32130 |     ms |
+|                                        90th percentile latency |        multimodal-search |    38269.2 |     ms |
+|                                        99th percentile latency |        multimodal-search |    39647.9 |     ms |
+|                                       100th percentile latency |        multimodal-search |      39785 |     ms |
+|                                   50th percentile service time |        multimodal-search |    229.561 |     ms |
+|                                   90th percentile service time |        multimodal-search |    267.099 |     ms |
+|                                   99th percentile service time |        multimodal-search |    494.745 |     ms |
+|                                  100th percentile service time |        multimodal-search |    496.515 |     ms |
+|                                                     error rate |        multimodal-search |          0 |      % |
+
+
+----------------------------------
+[INFO] SUCCESS (took 1058 seconds)
+----------------------------------
+```
+
+### Gotchas
+1. The above benchmark is running against a cluster that has two data nodes (See Docker compose [detail](https://docs.opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/#sample-docker-composeyml)). 
+If your cluster only have one data node, test procedures may stuck in `check-cluster-health` step, in that case, you should add a `number_of_replicas` parameter with value `0`
```

Comment:
only has a single data node, test procedures may get stuck in the `check-cluster-health` step.  In that case